charts/rancher-monitoring/v0.0.4/values.yaml

```diff
@@ -366,6 +369,11 @@ prometheus:
     clusterDisplayName: ""
   cluster:
     alertManagerNamespace: ""
+  thanos:
```

Comment:
Are the duplicate repository and tag settings required for some reason?

Reply:
That was a mistake. I will correct.

Reply:
Fixed.